src/plugins/utils/LambdaEventUtils.ts

```diff
@@ -360,6 +363,32 @@ class LambdaEventUtils {
         return className;
     }
 
+    static injectTriggerTagsForNetlify(span: ThundraSpan, pluginContext: any, originalContext: any): string {
+        const className = ClassNames.NETLIFY;
+        const domainName = DomainNames.API;
+        let operationName = 'netlify-site';
+
+        const siteName = process.env[NetlifyConstants.NETLIFY_SITE_NAME];
+
+        if (siteName) {
+            operationName = siteName;
+        }
+
+        const originalHandler = process.env[EnvVariableKeys._HANDLER];
+        const functionName = operationName + '/' + originalHandler.substring(0, originalHandler.indexOf('.'));
+
+        span._setOperationName(functionName);
+        originalContext.functionName = functionName;
```

Comment:
Might there be a side effect as we are mutating original context property? Might there be another way?

Reply:
Is it just needed to be used while getting application id over `LambdaUtils.getApplicationId`?